services/tables/src/main/java/com/linkedin/openhouse/tables/api/spec/v0/response/GetAllSoftDeletedTablesResponseBody.java

```diff
@@ -0,0 +1,23 @@
+package com.linkedin.openhouse.tables.api.spec.v0.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.gson.Gson;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.Value;
+import org.springframework.data.domain.Page;
+
+@Builder
+@Value
+public class GetAllSoftDeletedTablesResponseBody {
+
+  @Schema(
+      description = "Page of soft deleted tables matching the filters by database",
+      example = "")
+  @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+  private Page<GetSoftDeletedTableResponseBody> pageResults;
```

Comment:
Make sense to have paginated response to get list of all soft deleted tables under a DB.